test/integration/controller/jobs/job/job_controller_test.go

```diff
@@ -2004,35 +2011,9 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler when waitForP
 				g.Expect(*createdJob.Status.Ready).Should(gomega.Equal(int32(1)))
 				g.Expect(k8sClient.Get(ctx, wlKey, wl)).Should(gomega.Succeed())
 				g.Expect(wl.Status.RequeueState).ShouldNot(gomega.BeNil())
-				// Assert count>0 since the workload may get requeued and evicted
```

Comment:
I think now we can also remove the section above:
```
ginkgo.By("setting job status ready")
```
Right?

Reply:
Oh yes, you right, it's not need.